mobile/src/screens/auth/components/label/styles.ts

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+
+import { AppColor } from '#libs/enums/enums';
+
+const styles = StyleSheet.create({
+  labelContainer: {
```

Comment:
Why do we need this wrapper? for me from design it looks like usual text, so that we can get read of label component and at it to input it self and pass only text value